Add Vantiv to Payments section

oliver-donorschoose opened the pull request:
Vantiv is one of the biggest payment processors on the planet. Every year, they process >20B billion transactions and $726B in volume. Unfortunately, their merchant-facing console/portal does not offer 2FA.

oliver-donorschoose committed:
Add icon for Vantiv, a new addition to Payments

Vantiv is one of the biggest global payment processors on the planet.

oliver-donorschoose committed:
Add Vantiv to Payments section

Vantiv is one of the biggest payment processors on the planet. Every year, they process >20B billion transactions and $726B in volume. Unfortunately, their merchant-facing console/portal does not offer 2FA.

Jawshy commented on _data/payments.yml:
```diff
@@ -135,6 +135,12 @@ websites:
+      twitter: Vantiv
+      img: vantiv.png
+      tfa: No
+      
```
Please remove these blank spaces.

Jawshy reviewed (changes requested):
Thanks for submitting a pull request, @oliver-donorschoose. 

Could you please add their Facebook page? 
```ruby
      facebook: Vantiv
```

[Travis CI](https://travis-ci.org/2factorauth/twofactorauth/jobs/182438994): 
```
1. img/payments/vantiv.png is not 32x32 pixels.
2. img/payments/vantiv.png should not be larger than 2500 bytes. It is currently 24786 bytes.
```

Would you also be able to edit the image so that it is both 32 by 32 pixels and under 2,500 bytes? You can use tinypng.com to compress the image once it has been resized.

oliver-donorschoose committed:
Add Vantiv to Payments section

Vantiv is one of the biggest payment processors on the planet. Every year, they process >20B billion transactions and $726B in volume. Unfortunately, their merchant-facing console/portal does not offer 2FA.

oliver-donorschoose committed:
Icon for Vantiv, new addition to Payments

stephengroat committed:
cleanup whitespace

Jawshy committed:
Payments: Vantiv: Updated: Icon
